pkgs/development/python-modules/torchvision/bin.nix

```diff
@@ -9,13 +9,13 @@
 , pillow
 , python
 , pytorch-bin
+, version ? "0.10.0"
```

Comment:
Please revert this change.

Reply:
I can revert it, but it doesn't make sense to add version to `import ./binary-hashes.nix version`.

Reply:
@SuperSandro2000 
If we want torchvision-bin to support both lts and the latest, should I make torchvision-lts-bin and pytorch-lts-bin?